src/App/components/Tasks.js

```diff
@@ -0,0 +1,74 @@
+import React, { memo, useContext } from 'react';
+import styled from 'styled-components';
+import { TaskContext } from '../contexts/taskContext';
+import { DeleteButton } from './DeleteBtn';
+import RoundCheckBox from './Checkbox';
+
+const ListItem = styled.li`
+  position: relative;
+  font-size: 24px;
+  border-bottom: 1px solid #ededed;
+
+  &:hover .DeleteBtn {
+      display: block;
+      cursor: pointer;
+  };
+
+  &.editing:last-child {
+    margin-bottom: -1px;
+  }
+
+  &.editing {
+    border-bottom: none;
+    padding: 0;
+  }
+
+  &.editing .edit{
+    display: block;
+    width: 90%;
+    padding: 12px 16px;
+    margin: 0 0 0 10%;
+  }
+
+  &.completed .Data{
+    color: #d9d9d9;
+    text-decoration: line-through;
+  }
```

Comment:
What's `.Data`?

Reply:
I renamed this

Reply:
Same as above Please do not code this way.

Reply:
At the moment, I'm working on completing all the function of the todo app. I will refactor the code according to your review once all functions are executable